src/Elm/Declare.elm

```diff
@@ -299,14 +414,15 @@ value name expression =
     let
         declaration_ :
             { declaration : Elm.Declaration
-            , call : List Elm.Expression -> Elm.Expression
-            , callFrom : List String -> List Elm.Expression -> Elm.Expression
+            , call : List String -> List Elm.Expression -> Elm.Expression
```

Comment:
Would be nice if `value` also returned a `Decl`?

Reply:
Done!